mautic/mautic
# Fix prevent save alias of contact field already used in segments

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for all features, enhancements and bug fixes (until 3.3.0 is released) 
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 



#### Description:
Custom field alias generation was using reserved words, This PR prevents it.



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create custom field with company_id or contact_id alias
3. Submit
4. Check if the field is created (You should have an error)
5. Create custom field with random alias 
6. Submit
7. Check if field is created (You should have your field created)




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10224"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
